cmd/cloud.go

```diff
@@ -308,7 +315,67 @@ func cloudCmdFlagSet() *pflag.FlagSet {
 	return flags
 }
 
+//nolint:gochecknoglobals
+var cloudLogsCmd = &cobra.Command{
+	Use:   "logs",
+	Short: "Tail cloud logs from the cloud",
+	Long: `Tail cloud logs from the cloud.
+This will execute the test on the Load Impact cloud service. Use "k6 login cloud" to authenticate.`,
+	Example: `
+        k6 cloud logs 123456`[1:],
```

Comment:
- I'm wondering whether this makes sense as a sub-command. It would seem more intuitive if it was a `--logs` or `--tail-logs` or similar flag for the `k6 cloud` command instead, since it should only toggle the tail behavior.
- Is this example wrong? How can this command execute a new test if it asks for a test ID? Is this meant to run in a separate terminal after `k6 cloud` is run? The asciinema you showed last week did this in a single command, so I'm a bit confused by this documentation.

Reply:
Sorry, the description is wrong. This will NOT execute the test, this will get the logs of an already executed/executing test in the cloud and just tail them on the screen, the same way the original k6cloudlogs does

Reply:
I see. And it's a separate command because we want to enable retrieving logs for past runs as well. Makes sense then, but what do you think about integrating it in `k6 cloud` itself, like you showed in the asciinema? It could be disabled by default and enabled with a flag.

Reply:
it is currently [enabled by default](https://github.com/loadimpact/k6/pull/1599/files#diff-cf75533afdaed357cb53a3fdfe363a99R251-R256), and there is no flag ... and I would argue it should be disabled by a flag not enabled by it.